### PR TITLE
Fixes #246: Disable auto showing up the soft keyboard in login activity

### LINKED ISSUE #246: Don't open the keyboard immediately on email/password login page
Since on Android, Google auth is usually going to be what folks want, we should probably not open the keyboard immediately (since it sorta obscures the Google auth button) and require users to click into the text boxes to start typing.  What do you think @kunall17?

### PR DESCRIPTION
**Pull request by saketkumar95**
Fixes: [#246](https://github.com/zulip/zulip-android/issues/246)

@kunall17 Now keyboard shows up in the first screen for server URL and don't shows UP for login Activity.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@saketkumar95, it looks like you've already signed the Dropbox CLA. Thanks!

File: app/build.gradle [hunk 1]
```diff
@@ -43,7 +43,7 @@ android {
             applicationIdSuffix '.dev'
             manifestPlaceholders = [ fabricKey : ""]
             ext.enableCrashlytics = false
-            buildConfigField "String", "GOOGLE_CLIENT_ID", "\"659545034376-78hg5hmrvqr50nn53t0nme97ndhqjmvs.apps.googleusercontent.com\""
+            buildConfigField "String", "GOOGLE_CLIENT_ID", "\"835904834568-77mtr5mtmpgspj9b051del9i9r5t4g4n.apps.googleusercontent.com\""
```
Comment (kunall17) [inline app/build.gradle]: Not to be included in this commit!

**Comment by kunall17** [comment]
This doesn't do the things we intend to, we want something in which the keyboard shows at the first screen when it asks for the server URL and after when the backends are fetched and the login controls are shown then the keyboard should be hidden! 
Also remove windowSoftInputMode from many other activities!

**Comment by saketkumar95** [comment]
@kunall17 I will check it again and update the branch.. :)

**Comment by saketkumar95** [comment]
@kunall17 Done! :)

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -41,17 +41,16 @@
...
         <activity
             android:name=".activities.LoginActivity"
-            android:label="@string/app_name"
-            android:windowSoftInputMode="stateHidden" >
+            android:label="@string/app_name">
         </activity>
         <activity
             android:name=".activities.LegalActivity"
-            android:label="@string/title_activity_legal" >
+            android:label="@string/title_activity_legal">
```
Comment (kunall17) [inline app/src/main/AndroidManifest.xml]: Remove these formatting changes

**Comment by saketkumar95** [comment]
@kunall17 Done! :)

**Comment by kunall17** [comment]
- Add android:windowSoftInputMode="stateVisible" to the LoginActivity 
- Squash the commits

File: app/src/main/java/com/zulip/android/activities/LoginActivity.java [hunk 3]
```diff
@@ -222,6 +225,13 @@ private void showBackends(String httpScheme, String serverURL) {
                     @Override
                     public void onSuccess(Call<ZulipBackendResponse> call, Response<ZulipBackendResponse> response) {
                         if (response.body().isPassword()) {
+
```
Comment (kunall17) [inline app/src/main/java/com/zulip/android/activities/LoginActivity.java]: This hiding of keyboard should be outside isPassword if condition, it's not necessary every server supports EmailBackend!

**Comment by saketkumar95** [comment]
@kunall17 Closing this PR. As i'm having some problems in squashing the commits in master branch.
Created the new PR [#278](https://github.com/zulip/zulip-android/pull/278) here

**Pull request closed without merging**